android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt

```diff
@@ -0,0 +1,49 @@
+package com.swmansion.rnscreens
+
+import android.view.ViewGroup
+import androidx.annotation.UiThread
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.bridge.WritableNativeMap
+import com.facebook.react.uimanager.FabricViewStateManager
+import com.facebook.react.uimanager.PixelUtil
+import kotlin.math.abs
+
+abstract class FabricEnabledViewGroup constructor(context: ReactContext?) : ViewGroup(context), FabricViewStateManager.HasFabricViewStateManager {
```

Comment:
I think this name can be misleading.
What we do here is not enabling Fabric, but adding state. Could we change it to sth like `StateEnableScreenViewGroup`?

Reply:
Or `SizeAdjustingViewGroup` if we have different implementations for fabric and paper